templates/admin/reload_tool.mako

```diff
@@ -14,6 +14,17 @@ $().ready(function() {
 %endif
     focus_el.focus();
 });
+$().ready(function() {
+    $("#reload_toolbox").click(function(){
+        $.ajax({
+            url: "${h.url_for(controller="/api/configuration", action="toolbox")}",
+            type: 'PUT',
+            success: function(result) {
+                alert("Toolbox reloaded.");
```

Comment:
Can we please do alert when this fails instead? Or maybe just a console.log (in that case it can log both situations)